manifests/fedora-coreos.yaml

```
@@ -91,6 +91,19 @@ postprocess:
     cat /usr/etc/rpm-ostreed.conf >> /tmp/rpm-ostreed.conf
     cp /tmp/rpm-ostreed.conf /usr/etc/rpm-ostreed.conf
     rm /tmp/rpm-ostreed.conf
+  - |
+    #!/bin/bash
+    find /etc/ /usr/ -type l | while read file_name; do     
```

White space at the end of lines is sometimes significant in Bash script. Let's try not to add some wen it's not needed:

```suggestion
    find /etc/ /usr/ -type l | while read file_name; do
```

This also applies to the line below